runtime/gc_modron_standard/StandardAccessBarrier.cpp

```diff
@@ -748,6 +748,9 @@ MM_StandardAccessBarrier::preMonitorTableSlotRead(J9VMThread *vmThread, j9object
 {
 	omrobjectptr_t object = (omrobjectptr_t)*srcAddress;
 
+	if (NULL == _extensions->scavenger) { /* This check is needed if gc_alwaysCallObjectAccessBarrier build flag is enabled */
+		return true;
+	}
```

Comment:
remove these changes. they are correct, but we'll consider them as part of another PR